src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -619,16 +619,14 @@ private List<Integer> getEmptyLines(DetailAST ast) {
      * @param emptyLines list of empty lines.
      * @return list of empty lines to log.
      */
-    private static List<Integer> getEmptyLinesToLog(List<Integer> emptyLines) {
+    private static List<Integer> getEmptyLinesToLog(Iterable<Integer> emptyLines) {
```

Comment:
https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=12734&view=logs&j=eb841e9f-6e1e-5c29-3ad5-74ec2c658069&t=a9b761e6-b8d5-50cf-467a-54967189cab2&l=1262
PMD said this had to change.

Reply:
This link isn't working for me, is it related to removing the call to `size()` and being able to use an interface higher in the inheritance chain?

Reply:
> is it related to removing the call to size() and get() and being able to use an interface higher in the inheritance chain?

Yes, it is similar to if I use `ArrayList` as a type, but could downgrade it to `List` or even further as `Collection`.